redisinsight/api/src/modules/database-recommendation/repositories/local.database.recommendation.repository.ts

```diff
@@ -33,6 +35,8 @@ export class LocalDatabaseRecommendationRepository extends DatabaseRecommendatio
     @InjectRepository(DatabaseRecommendationEntity)
     private readonly repository: Repository<DatabaseRecommendationEntity>,
     private eventEmitter: EventEmitter2,
+    private readonly analytics: DatabaseRecommendationAnalytics,
+    private readonly databaseService: DatabaseService,
```

Comment:
We should not inject this here. It is better to move it to upper layer so if at some point we will add another repository analytics will still work

Reply:
moved